core/conversion/evaluators/prim.cpp

```diff
@@ -32,6 +32,12 @@ auto prim_registrations =
                     [](const torch::jit::Node* n, kwargs& args) -> c10::optional<torch::jit::IValue> {
                       return at::scalar_to_tensor(args.at(n->output(0)).IValue()->toScalar());
                     }})
+        .evaluator({torch::jit::prim::ListUnpack,
+                    [](const torch::jit::Node* n, kwargs& args) -> c10::optional<torch::jit::IValue> {
+                      // Outputs is an IValue which has list of tensors which can be found in ctx->evaluated_value_map
+                      const torch::jit::IValue* outputs = args.at(n->input()).IValue();
+                      return *std::move(outputs);
```

Comment:
You might be able to use something like this to quickly switch to a tuple https://github.com/pytorch/pytorch/blob/22a34bcf4e5eaa348f0117c414c3dd760ec64b13/test/cpp/api/jit.cpp#L91

Reply:
Done